### ISSUE TITLE
Add a brief description for each AI to its page in the Web site

### ISSUE THREAD

**Issue by davidbrucesmith** [CHI-CityTech/Balanced-Blended-Space#2]
On [this page](https://sites.google.com/view/balancedblendedspace/projects/ai-image-generators-evaluation), add a brief description for each AI source that we used.  A simple sentence or two that describe the site, its background, or what it specializes in would be helpful.  It can be on either the main image under the site logo, or it could be at the top of the drop down information.

**Comment by pbgm5** [comment]
Priya Comment:

Added descriptions under each website logo, issues I came across when making a "brief description" was when it got too wordy and long, the spacing of everything made it look bad, I resolved this issue by using and editing empty spacing blocks but spacing is a nightmare to perfect. I researched each websites developer name and release date, it began to waste a lot of time looking through each website on google so I turned to Chat GPT for faster answers. Another issue I came across was as I kept making each description for the less popular sites, the descriptions became more bare boned compared to the unique or popular websites.
 
- Fixed Adobe FireFly website link, after realizing it has it's own standalone image generating website

- Realized too late "HP Copilot" instead of "Microsoft Copilot" on website so I fixed the name.

- Fixed the Copilot website link (incorrect link)

- Added a text box on top of the two prompt images of Copilot, where it didn't let me generate prompt one so viewers don't get confused as to why Prompt 1 image is missing for Copilot. 

-  Apparently AI test kitchen changed their name, logo and website link so I changed all of it to stay up to date on the website

Questions: 
Ask Professor if the tests are valid since some use the same models such as Copilot using DALL-E

This took me 3 and half hours total.

**Comment by davidbrucesmith** [comment]
Investigate differetn layouts in teh Copy of the page, and then update to what looks best for you

**Comment by pbgm5** [comment]
Replaced all the hyperlinks, and implemented the website links into the name of the generator itself to save space. Changed image font to blue so users know it's a clickable link for each hyperlink. took sometime experimenting how to format large text properly. This all took approx. 1 hour.
